spec/game_statistics_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe GameStatistics do
+    before(:each) do
+        game_path = './data/games_dummy.csv'
+
+        @game_stats = GameStatistics.new(StatTracker.game_reader(game_path))
```

Comment:
This is still confusing. We will need to revisit this logic before our presentation to make sure we can all describe it adequately.